### DISCUSSION TITLE
how can i set png dpi metadata

### DISCUSSION THREAD

**Discussion by srxqds** [SixLabors/ImageSharp#2390] [Q&A]
how can i set png dpi metadata？

**Comment by JimBobSquarePants** [comment]
What do you mean my this? 

Do you mean the [DPI](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Metadata.ImageMetadata.html#SixLabors_ImageSharp_Metadata_ImageMetadata_HorizontalResolution) of the image (this property is non-format specific)

**Reply by srxqds** [reply]
> What do you mean my this?
> 
> Do you mean the [DPI](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.Metadata.ImageMetadata.html#SixLabors_ImageSharp_Metadata_ImageMetadata_HorizontalResolution) of the image (this property is non-format specific)

yes, how can I set it value?

**Reply by JimBobSquarePants** [reply]
Using the property I linked in addition to the `VerticalResolution` property.
